.github/workflows/build-images.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           tags: chromatichq/${{ matrix.service }}:${{ matrix.php }}
 
       - name: Notify slack on failure
-        if: failure()
+        if: failure() && github.event_name != 'pull_request'
```

Comment:
This will still be triggered if the workflow is run manually via workflow dispatch. Maybe we switch it to check for push?

Reply:
I figured we would _want_ to report a failure on workflow dispatch. If that’s not the case, then I’m happy to change this to check for push explicitly.

Reply:
Well, the person who runs it already gets an email about the failure even if they aren't sitting there and watching it. For me, workflow_dispatch is often used here when troubleshooting so I wouldn't want those failures to spam everyone in Slack.

Reply:
That makes sense! I updated both steps to report failures on both the `push` and `schedule` events in 11eb293. Let me know what you think.

Reply:
💯